go/common/logging/logging.go

```diff
@@ -213,6 +213,19 @@ func GetLoggerEx(module string, extraUnwind int) *Logger {
 	return backend.getLogger(module, extraUnwind)
 }
 
+// GetBaseLogger creates a new non-prefixed logger instance with the
+// specified module.
+//
+// The returned logger will not pre-include any log fields (aka prefixes)
+// except for the module name. Its level will be set in accordance
+// with the global config, which can be per-module.
+//
+// This may be called from any point, including before Initialize is
```

Comment:
I implemented this (and copy-pasted the doc paragraph) for consistency with GetLogger(), but I'm happy to simplify the code and panic if logging is not initialized and the method is called. It would work for the current use case, and no other use cases are foreseen.